### ISSUE TITLE
log file

### ISSUE THREAD

**Issue by Wang-pengfei** [NVlabs/FocalFormer3D#27]
Hi, I'm trying to reproduce FocalFormer3D-LC, but I seem to be encountering a strange issue. I was wondering if you still have the previous training log file? It doesn't have to be an exact replica of the model mentioned in the paper; I just want to compare the log files to identify any potential issues. Thank you.

**Comment by chenyilun95** [comment]
Thanks for your attention. However, I did not save the log file on my current machine. Let me know if anything strange here.

**Comment by Wang-pengfei** [comment]
Thanks for your prompt response! I achieved 70.13 mAP and 71.94 NDS on the validation set in the first epoch, and 70.21 mAP and 71.57 NDS in the second epoch. This seems a bit unusual, and there's a significant difference compared to the final performance of 70.5 mAP and 73.1 NDS. Is this normal? or can I expect the performance to improve in the sixth epoch?

**Comment by chenyilun95** [comment]
I did not specifically check the performance of epoch 1. It should improve in the following epochs, as the copy-paste augmentation is closed after epoch 1. You can refer to the PointAugment paper for details on performance changes.

**Comment by Wang-pengfei** [comment]
I couldn't find the "fade_epoch" setting in the FocalFormer3D_LC.py file. Is this normal, or was it accidentally omitted?

**Comment by chenyilun95** [comment]
Sorry I mistook the fusion model config. It's not necessary to set fade_epoch, as the image did not use it in this code. So you need to wait and check for the last performance.

**Comment by Wang-pengfei** [comment]
Okay, thank you for your patient answers. I will wait and see how the final results turn out.    
Wishing you a pleasant weekend!

**Comment by WenxuanLi-whu** [comment]
能不能发个训练日志到我地邮箱一下：
liwenx@whu.edu.cn
